packages/graphql/schemas/schema.graphql

```diff
@@ -1596,6 +1599,15 @@ type ReactComponentDescriptor {
   isDefault: Boolean!
 }
 
+"""Response from getReactComponentsFromFile"""
+type ReactComponentResponse {
+  """Components that are exported from the parsed file"""
+  components: [ReactComponentDescriptor!]!
+
+  """Whether or not there was there an error when parsing the file"""
+  errored: Boolean
```

Comment:
Will we care about the type of error/message?

Reply:
we don't display the actual error in the UI, so no. The full error will be logged via debug https://github.com/cypress-io/cypress/pull/24982/files#diff-b24c7f86b6dcda7a4a5727db3592d6a026e3334843cf65aba4313189885497bfR47